src/components/Sidebar/Properties/PropertyOwnerHeader.jsx

```diff
@@ -101,6 +182,11 @@ function PropertyOwnerHeader({
 
   const hasMoreButtons = (popOutAction || metaAction);
 
+  let checkBoxFadeStyle = {}
+  if (fadeUri && fadeValue > 0) {
+    checkBoxFadeStyle = {opacity: fadeValue};
```

Comment:
Just a thought: maybe we could interpolate the fade value on the GUI side to make it smoother, instead of using the property? (Feel free to ignore)

Reply:
Thought about it too, and even if it would look nicer it does mean that I would have to keep a state around just for the fading. Also, unfortunately, I don't think I would have time to look at it before the code freeze

But let's leave it as a future improvement